tests/unit/parser/TerraformParser.spec.js

```diff
@@ -0,0 +1,331 @@
+                }),
+                new ComponentAttribute({
+                  name: 'most_recent',
+                  value: true,
+                  type: 'Boolean',
+                }),
+              ],
+            }),
+            new Component({
+              name: 'publicdns',
+              id: 'publicdns',
+              definition: new TerraformComponentDefinition({
+                blockType: 'resource',
+                provider: 'aws',
+                type: 'aws_route53_zone',
+                icon: 'AwsRoute53Zone',
+                model: 'DefaultModel',
+                isContainer: true,
+                definedAttributes: [new ComponentAttributeDefinition({
+                  name: 'name',
+                  type: 'String',
+                  required: true,
+                })],
+              }),
+              attributes: [
+                new ComponentAttribute({
+                  name: 'name',
+                  value: 'aws.domaine.fr',
+                  type: 'String',
+                  definition: new ComponentAttributeDefinition({
+                    name: 'name',
+                    type: 'String',
+                    required: true,
+                  }),
+                }),
+              ],
+            }),
+            new Component({
+              id: 'image_id',
+              name: 'image_id',
+              definition: new TerraformComponentDefinition({
+                blockType: 'variable',
+                provider: 'aws',
+                type: 'image_id',
+                icon: 'ImageId',
+                model: 'DefaultModel',
+              }),
+              attributes: [new ComponentAttribute({
+                name: 'type',
+                type: 'String',
+                value: 'string',
+              })],
+            }),
+          ]);
+        });
+      });
+
+      describe('Test parse: container', () => {
+        const definitions = new TerraformMetadata({
+          metadata: {
+            aws: JSON.parse(fs.readFileSync('tests/resources/metadata/container.json', 'utf8')),
+          },
+        }).getDefinitions();
+        const parser = new TerraformParser(definitions);
+
+        it('expect to have valid tree', () => {
+          const input = fs.readFileSync('tests/resources/tf/container.tf', 'utf8');
+          const { components } = parser.parse([input]);
+
+          expect(components.length).toEqual(1);
+          expect(components[0].id).toEqual('parent');
+
+          expect(components[0].children.length).toEqual(1);
+          expect(components[0].children[0].id).toEqual('child');
+        });
+      });
+
+      describe('Test parse: links', () => {
+        const definitions = new TerraformMetadata({
+          metadata: {
+            aws: JSON.parse(fs.readFileSync('tests/resources/tf/link.json', 'utf8')),
+          },
+        }).getDefinitions();
+        const parser = new TerraformParser(definitions);
+
+        it('test single default link', () => {
```

Comment:
From here, the `it()` (all of them) should contain sentence which explains the verification (like above).
For instance:
`it("should parse a default link")`